### PR TITLE
[TEL-270] Show pre-call information in the iOS app

### PR DESCRIPTION
**Pull request by ivanjiang5628**
## Description
- [Jira Ticket](https://janeapp.atlassian.net/browse/TEL-270?atlOrigin=eyJpIjoiYjM3NmQwZmM2YjBiNGEzN2IyZGQzNmMwMDg4MWQwMDQiLCJwIjoiaiJ9)

- add a new `waitingMessage` component for showing the pre-call information

### Risk / General PR Class
- [x] 💚 = Low Risk

🌟 = New Feature (Adds Functionality)
☕️ = JS Dependency Changes

### Demo Notes

Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-07-29 at 11 28 33

Simulator Screen Shot - iPhone 11 Pro Max - 2020-07-29 at 11 27 19

## Code Review
Resource: [Dev Team Notion Page](https://www.notion.so/janeapp/Dev-Team-f06c6eb2ccca4066bc63fc1ac1bd2549)
Resource: [Code Review Checklist](https://www.notion.so/janeapp/Code-Review-checklist-2c510c527ac7470c902a5e8f25f9db3c)

- [x] I clearly explained the WHY behind the work, in the Description above

#### Design
- [x] I added instructions for how to test, in the QA section below
- [ ] I added tests for changes, or determined that none were required
- [ ] I demoed this to the appropriate person
- [x] I considered both mobile & desktop views, or that wasn't relevant

#### Code
- [x] I committed code with informative git messages
- [x] I wrote readable code, or added comments if it was complex
- [x] I performed a self-review of my own code
- [x] I rebased my branch on the latest `master`
- [x] I confirmed that all CircleCI tests are passing
- [x] I didn't add new npm packages, or else I made damn sure the `yarn.lock` changes are safe for existing production packages

### CODE REVIEW

File: react/features/large-video/components/LargeVideo.native.js [hunk 1]
```diff
@@ -8,7 +8,9 @@ import { connect } from '../../base/redux';
 import { DimensionsDetector } from '../../base/responsive-ui';
 import { StyleType } from '../../base/styles';
 
+import WaitingMessage from '../../base/react/components/native/WaitingMessage.js'
 import { AVATAR_SIZE } from './styles';
+import { Text, View } from 'react-native';
```
Comment (gerbus) [inline react/features/large-video/components/LargeVideo.native.js:13]: I think you can remove these, they don't look to be used here.

**Review by gerbus — APPROVED** [review]
Looks good! I did find two minor things but nothing critical enough to not approve.

File: react/features/base/react/components/native/styles.js [hunk 2]
```diff
@@ -203,6 +203,43 @@ const SECTION_LIST_STYLES = {
...
+    waitingMessageHeader: {
+        fontSize: 15,
+        marginTop: 5,
+        color: ColorPalette.janeDarkColor,
+        textAlign: 'left',
+        backgroundColor: 'transparent'
+    },
+
+    waitingMessageText: {
+        marginTop: 5,
```
[the remark this answers is not in the document]
Comment (ivanjiang5628) [reply react/features/base/react/components/native/styles.js:228]: I agree I will add a Jira backlog ticket later👍

File: react/features/base/react/components/native/WaitingMessage.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        ).start(() => this._animate())
+    }
+
+    _startTimer() {
+        const { jwt, conferenceHasStarted } = this.props;
+        const jwtPayload = jwt && jwtDecode(jwt);
+        if (jwtPayload && jwtPayload.context && !conferenceHasStarted) {
+            const { start_at } = jwtPayload.context || 0;
+            // const appointmentStartTimeStamp = new Date(start_at).getTime();
+            const appointmentStartTimeStamp = moment(moment(start_at, 'YYYY-MM-DD HH:mm:ss')).valueOf()
```
[the remark this answers is not in the document]
Comment (ivanjiang5628) [reply react/features/base/react/components/native/WaitingMessage.js:66]: Good catch! I will fix it.

**Comment by ivanjiang5628** [comment]
Close this PR because this feature is included in [TEL-281](https://github.com/janeapp/video-chat/pull/18) PR.

**Pull request closed without merging**